opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/ListTaskInteractor.java

```diff
@@ -186,9 +201,165 @@ private JSONObject createFeatureCollection() {
         return featureCollection;
     }
 
-    private ListTaskContract.Presenter getPresenter() {
-        return (ListTaskContract.Presenter) presenterCallBack;
+    public void saveJsonForm(String json) {
```

Comment:
Remove this method, method already defined in the parent class `BaseInteractor`. Saving should be on `BaseInterator` so that it be called from task register presenter and family profile presenter

Reply:
Done.